### PR TITLE
Added Monaco holidays

### PR DESCRIPTION
**Pull request by KJhellico**

### CODE REVIEW

File: holidays/countries/monaco.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    def _populate(self, year):
+        def _add_observed(hol_date: date) -> None:
+            if self.observed and hol_date.weekday() == SUN:
+                self[hol_date + rd(days=+1)] = f"{self[hol_date]} (Observed)"
+
+        super()._populate(year)
+
+        # New Year's Day
+        dt = date(year, JAN, 1)
+        self[dt] = "Le jour de l’An [New Year's Day]"
```
Comment (arkid15r) [inline holidays/countries/monaco.py:41]: It seems ’ is quite stubborn one. Let's replace them with ' again (line 41, 58, 87)!

**Review by arkid15r — COMMENTED** [review]
🇲🇨 LGTM!

File: holidays/countries/monaco.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/monaco.py]: Done. Thank you!

**Comment by arkid15r** [comment]
Thanks for adding one more country, @KJhellico!

**Pull request merged**